packages/vats/src/proposals/upgrade-agoricNames-proposal.js

```diff
@@ -43,5 +43,8 @@ export const getManifestForUpgradingAgoricNames = (
       produce: {},
     },
   },
+  installations: {
+    walletFactory: restoreRef(agoricNamesRef),
+  },
```

Comment:
`walletFactory`? I suppose that's a copy-and-paste-o

but more to the point: agoricNames is not a contract vat. There's no relevant installation.